Dockerfile

```diff
@@ -68,7 +69,12 @@ RUN xx-apt-get install -y binutils gcc libc6-dev libgcc-10-dev libsecret-1-dev p
 
 FROM base AS test
 ARG DEBIAN_FRONTEND
+ARG GOPASS_VERSION
 RUN xx-apt-get install -y dbus-x11 gnome-keyring gpg-agent gpgconf libsecret-1-dev pass
+RUN --mount=type=bind,target=. \
+    --mount=type=cache,target=/root/.cache \
+    --mount=type=cache,target=/go/pkg/mod \
+    go install github.com/gopasspw/gopass@${GOPASS_VERSION}
```

Comment:
```suggestion
    GOFLAGS= go install github.com/gopasspw/gopass@${GOPASS_VERSION}
```

Reply:
yep, just saw the error myself and followed the ~onion~ stages down to understand why the sandbox stage failed.